packages/react-topology/src/components/TopologyView/examples/topology-example.css

```diff
@@ -1,11 +1,11 @@
-.ws-react-t-baseline,
 .ws-react-t-control-bar,
 .ws-react-t-custom-edges,
 .ws-react-t-custom-nodes,
+.ws-react-t-getting-started,
 .ws-react-t-panzoom,
-.ws-react-t-selectable,
-.ws-react-t-with-sidebar,
-.ws-react-t-with-toolbar {
+.ws-react-t-selection,
+.ws-react-t-sidebar,
```

Comment:
For the styles just below these, looks we want to also rename the sidebar class since it looks like we lost the drawer styling. e.g.:

```
.ws-react-t-sidebar .topology-example-sidebar {
  max-width: 300px;
}
```